src/components/LoginForm.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { FormEvent, useContext, useState } from 'react';
+import Cookies from 'js-cookie';
+import {
+  Box,
+  Button,
+  Form,
+  FormField,
+  Heading,
+  HelperErrorText,
+  TextInput,
+} from '@nypl/design-system-react-components';
+import { TokenContext, TokenContextValues } from '../context/tokenContext';
+
+const backgroundStyles = {
+  background: 'section.blogs.primary',
+  margin: 'auto',
+  marginTop: 'xxl',
+  paddingBottom: 'xl',
+  paddingTop: 'l',
+  width: '400px',
+};
+const formFieldStyles = { margin: 'auto', width: '75%' };
+const errorStyles = {
+  color: 'ui.error.primary',
+  paddingTop: 'xs',
+  textAlign: 'center',
+};
+
+const LoginForm = () => {
+  const [username, setUsername] = useState<string>('');
+  const [password, setPassword] = useState<string>('');
+  const [showLoginError, setShowLoginError] = useState<boolean>(false);
+
+  const { setAccessToken } = useContext(TokenContext) as TokenContextValues;
+
+  const login = (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    const body = new FormData();
+    body.append('username', username);
+    body.append('password', password);
+
+    fetch('https://qa-libraryregistry.librarysimplified.org/admin/log_in/jwt', {
+      method: 'POST',
+      body,
+    })
+      .then((response) => {
+        if (response.ok) {
+          return response.json();
+        }
+        if (response.status >= 400 && response.status < 502) {
+          setShowLoginError(true);
+        }
+        throw new Error('There was a problem authenticating this user.');
+      })
+      .then((data) => {
+        setAccessToken(data.access_token);
```

Comment:
Do we also need to set the new refresh token value in the cookie? Or can the old refresh token be used more than once?